Source/Core/Core/PowerPC/Jit64Common/BlockCache.cpp

```diff
@@ -21,7 +21,7 @@ void JitBlockCache::WriteLinkBlock(const JitBlock::LinkData& source, const JitBl
 
   u8* location = source.exitPtrs;
   const u8* address = dest ? dest->checkedEntry : dispatcher;
-  Gen::XEmitter emit(location);
+  Gen::XEmitter emit(location, location + 32 /* FIXME */);
```

Comment:
This still needs to be addressed. Especially given there's no explanation to others about what actually needs to be fixed.

Reply:
Completely forgot about that one, thanks.